files/defaults.yaml

```diff
@@ -84,6 +84,10 @@ zowe:
       # Default to caching service entry as it predates this one
       name: "${{ ()=> { if (components['caching-service']?.storage?.vsam?.name) { return components['caching-service'].storage.vsam.name } else { return '' } }() }}"
 
+  network:
+    # This is a variable derived from gateway's ATTLS state to be used by servers proxied through it in eureka or static definitions.
+    proxyType: "${{ components.gateway?.zowe?.network?.server?.tls?.attls == true ? 'http' : zowe.network?.server?.tls?.attls == true ? 'http' : 'https' }}"
```

Comment:
Is this correct? `components.gateway?.zowe?.network?.server?.tls?.attls`
The `zowe` property under the `gateway`?

Reply:
yes, this property does exist.
"zowe.network" can be applied globally or per-component.
Prior, "zowe.network.server" and "zowe.network.client" had separate ATTLS properties.
But through discussions recently it appears that having client/server individually customizable for ATTLS is pointlessly complicated and so you should consider "zowe.network.client.tls.attls" to be deprecated and this is why this variable considers just 2 values: the server tls of the gateway config, and the server tls of the global config as a fallback if the gateway one is not set.